cpp/oneapi/dal/detail/error_messages.cpp

```diff
@@ -178,7 +178,7 @@ MSG(knn_search_task_is_not_implemented_for_gpu, "k-NN search task is not impleme
 MSG(neighbor_count_lt_one, "Neighbor count lower than one")
 MSG(unknown_distance_type,
     "Custom distances for k-NN is not supported, use one of the predefined distances instead.")
-MSG(distance_is_not_supported_for_gpu, "Only Euclidean distances for k-NN is supported for GPU")
+MSG(distance_is_not_supported_for_gpu, "Only Minkowsky distances for k-NN is supported for GPU")
```

Comment:
MSG(distance_is_not_supported_for_gpu, "Only Minkowsky distances for k-NN is supported for GPU")
```suggestion
MSG(distance_is_not_supported_for_gpu, "Only Minkowsky distances for k-NN are supported for GPU")
```